Uplift/Controllers/OnboardingGymsViewController.swift

```diff
@@ -228,8 +232,37 @@ class OnboardingGymsViewController: UIViewController, UITableViewDelegate, UITab
         navigationController?.popViewController(animated: true)
     }
 
-    // MARK: UITableViewDelegate Methods
+}
+
+extension OnboardingGymsViewController: UITableViewDataSource {
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        //swiftlint:disable:next force_cast
+        let cell = tableView.dequeueReusableCell(withIdentifier: FavoriteGymCell.reuseIdentifier, for: indexPath) as! FavoriteGymCell
+        let gym = gymNames[indexPath.section]
+        cell.configure(with: gym)
+        // Toggle if already in UserDefaults
+        if favoriteGyms.contains(gym) {
+            cell.toggleSelectedView(selected: true)
+        }
+        cell.setNeedsUpdateConstraints()
```

Comment:
we shouldn't need to call this. any update to constraints should just be done in the `configure` function